src/markdown-pages/instant-observability/build-a-quickstart/create-a-dashboard.mdx

```diff
@@ -0,0 +1,138 @@
+
+>```SQL
+>SELECT count(*) FROM fdb_method FACET method 
+>```
+
+![database methods chart](../../../images/instant-observability/fdb-methods.png)
+
+Here, you group your results of FlashDB methods by attribute values and see the count of each type of transaction. You can choose to present your results in different formats such as table, billboard, pie chart etc. 
+
+![chart formats](../../../images/instant-observability/chart-formats.png)
+
+Change the chart type to pie, name your chart and click save.
+
+![pie chart](../../../images/instant-observability/piechart.png)
+
+The chart is now visible on your dashboard.
+
+![piechart dashboard](../../../images/instant-observability/piechart-dashboard.png)
+
+
+You can add more charts to your dashboard following the same pattern. To do so, click **+** in upper right hand corner. 
+
+![add more charts](../../../images/instant-observability/add-more-charts.png)
+
+This will take you to the same _Add to your dashboard_ page, allowing you to add another chart, text, image or links. Add a few more charts to your dashboard as follow:
+
+### Average response time
+Run the following query to observe the **average response time** of database transactions. 
+
+>```SQL
+> SELECT average(fdb_create_responses), average(fdb_read_responses), average(fdb_update_responses), average(fdb_delete_responses) FROM Metric SINCE 30 MINUTES AGO TIMESERIES
+>```
+
+![average response time chart](../../../images/instant-observability/average-response-time.png )
+
+>**Note:** Make sure to click **Run** to see above results. 
+
+Here, you observe average response time for different transactions for past 30 minutes. Click save to add this chart to your dashboard as well. Follow the same procedure to add the following charts as well.
+
+### Errors
+
+>```SQL
+> SELECT sum(fdb_create_errors), sum(fdb_read_errors), sum(fdb_update_errors), sum(fdb_delete_errors) FROM Metric SINCE 60 MINUTES AGO TIMESERIES
+>```
+
+Here, you observe errors for all database transactions for past one hour.
+
+![errors chart](../../../images/instant-observability/errors.png )
+
+### Database size
+
+>```SQL
+> SELECT sum(mdb_size) FROM Metric
+>```
+
+![database size chart](../../../images/instant-observability/database-size.png )
+
+This chart will show you the database size. 
+
+### Cache hits
+
+>```SQL
+> SELECT sum(fdb_cache_hits) FROM Metric SINCE 30 MINUTES AGO TIMESERIES
+>```
+
+![cache hits chart](../../../images/instant-observability/cache-hits.png )
+
+Here, you observe the total number of cache hits for past 30 minutes using line chart.
+
+### Keys
+
+>```SQL
+> SELECT count(fdb_keys) FROM Metric SINCE 5 MINUTES AGO TIMESERIES
+>```
+
+![keys chart](../../../images/instant-observability/keys.png )
+
+Here, you observe the flashDB key count for past 5 minutes. 
+
+You can manipulate the above queries and change time window to focus on the data that you want to observe. Your final dashboard will look similar to the following:
```

Comment:
> You can manipulate the above queries and change time window to focus on the data that you want to observe.

Like we talked about before, for this content type, we actually prefer the reader use our exact queries so they can compare their code to ours in the end.